aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -75,6 +82,27 @@ ASP.NET Core project templates use Kestrel by default. In *Program.cs*, the temp
 
 [!code-csharp[](kestrel/samples/2.x/Program.cs?name=snippet_DefaultBuilder&highlight=7)]
 
+::: moniker range=">= aspnetcore-2.1"
+**Default transport changes from Libuv to managed sockets (ASP.NET Core 2.1 or later)**
+
+With the release of ASP.NET Core 2.1, Kestrel's default transport is no longer based on Libuv but instead based on managed sockets. This is a breaking change for ASP.NET Core 2.0 apps that call [WebHostBuilderLibuvExtensions.UseLibuv](/dotnet/api/microsoft.aspnetcore.hosting.webhostbuilderlibuvextensions.uselibuv) and depend on either of the following packages:
+
+* [Microsoft.AspNetCore.Server.Kestrel](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.Kestrel/)
+* [Microsoft.AspNetCore.App](https://www.nuget.org/packages/Microsoft.AspNetCore.App/)
+
+For ASP.NET Core 2.1 or later projects that require the use of Libuv (the app calls [WebHostBuilderLibuvExtensions.UseLibuv](/dotnet/api/microsoft.aspnetcore.hosting.webhostbuilderlibuvextensions.uselibuv)), update the project to directly reference the [Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/) package. Add a dependency for the Libuv transport package to the app's project file:
+
+```xml
+<PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" 
+                  Version="2.1.0" />
+```
+
+> [!NOTE]
+> The [Microsoft.AspNetCore.All](xref:fundamentals/metapackage) metapackage doesn't directly reference [Libuv](https://www.nuget.org/packages/Libuv/), but it does directly reference [Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/). The Libuv transport package is transitively referenced when the `Microsoft.AspNetCore.All` package is referenced. No action is required by the developer when the `Microsoft.AspNetCore.All` metapackage is used and `UseLibuv` is called in the app. Note that the ASP.NET Core 2.1 or later templates reference the [Microsoft.AspNetCore.App](https://www.nuget.org/packages/Microsoft.AspNetCore.App/) metapackage by default. The use of the `Microsoft.AspNetCore.All` metapackage isn't recommended for ASP.NET Core 2.1 or later apps.
```

Comment:
This is pretty wordy. I think your main point here is that the previous step is unnecessary in some cases, right? If so, consider shortening to

> No additional reference is required when using the Microsoft.AspNetCore.All metapackage, which will pull in Libuv transitively. For more details on this metapackage, see {cref}

Reply:
> main point here

It's there to explain why the `All` package doesn't require the dev to take action. I think we should be careful not to get too concise when providing technical details. It might be "wordy," but it's easy to read and understand because it doesn't try to say too much in a handful of words. However, I think we can drop the first sentence with a modification to the second sentence that will shorten this. Let's see how it looks on the next commit.